README.md

```diff
@@ -40,6 +40,13 @@ A demo version is automatically deployed from our repositories:
 
 The Open Event Server can be easily deployed on a variety of platforms. Detailed platform-specific installation instructions have been provided below.
 
+
+**NOTE :** If you are heading towards Local Installation, Be sure that you use one of the  following operating systems :
+
+
+* Debian based Linux Distribution
```

Comment:
@iamareebjamal Should we also add a recommendation for Debian version to be 10, as in the README, we have the command `sudo apt-get install python3.7` which will fail in Debian 9 as python3.7 is not available as a package in Debian 9.

Reply:
Most people use Ubuntu and don't know what's debian. So it should be `Debian based Linux Distros (like Ubuntu)` and no, we discourage using apt to install python because it will be always stuck with the distribution itself. Using `pyenv` is recommended as previously discussed. However, python installation is not the scope of this issue AFAIK

Reply:
@iamareebjamal Agree. @sanjana-302 Please make the following changes.

Reply:
@iamareebjamal @prateekj117 Thank you for suggestions. I will make mentioned changes.